Externals.cfg

```diff
@@ -34,5 +34,24 @@ hash = 394917e
 local_path = src/UPP
 required = True
 
+[GSI]
+protocol = git
+repo_url = https://github.com/NOAA-EMC/GSI.git
+# Specify either a branch name or a hash but not both.
+#branch = develop
+hash = a90b632
+local_path = src/gsi
+externals = None
```

Comment:
I guess the `externals = None` flag causes `manage_externals` to not check out any submodules.  But at some point when we want users to actually run gsi, I'm assuming this will have to be removed so that the submodules get recursively checked out, correct?  Will it take a long time to clone then?

Reply:
We will never turn on the checking out of GSI submodules.  The only submodule that is relevant is the one for the fix files, and those are currently handled in other ways by the RRFS team.  We can definitely discuss strategies for that.  But, in terms of submodules for GSI, they will never be checked out by `manage_externals` unless and until they are moved from VLab to some where that is accessible to the public at large.  It's not just a question of how long the checkout takes, it's also a question of access to VLab.  Currently, checking out the external submodules for GSI is a non-starter.

Reply:
Seems I'm missing something basic.  Why clone the top-level GSI repo if you can't clone its submodules?  Is it because those submodules (except possibly the one for fix files) are not necessary for running GSI?

Reply:
Correct.  The submodules are not needed.  One of them `libsrc` was removed some time ago because it has been superseded by use of hpc-stack libs.  The only one that remains is `fix`, and there are other ways of managing that.  The RRFS team's local GSL fork of GSI also does not include any git submodules and they are managing use of any required fix files in other ways.  We may need to get more information about GSI fix file requirements from the RRFS team and discuss how to handle any potential issues with that.  However, as far as the GSI repository is concerned, we absolutely cannot include checking out of VLab git submodules for the SRW App.  That is an absolute show stopper.  So, regarding this issue there are two options:

1. Check out GSI without the `fix` submodule
2. Don't add GSI to the SRW App and do not add RRFS capabilities to the SRW App.

Reply:
That's a great question.  I think I'll need to talk with the RRFS GSL folks to understand what they're doing before I can have a good answer to that.  I wanted to tag Ming here, but he doesn't appear to be on this repo, so I'll have to ping him offline.

Reply:
@gsketefian - What do you think about addressing the fix file issue in conjunction with adding the first RRFS workflow capability?  I think we've established that resolving handling of the fix files may take some time and involve some creative solutions.  My current favorite is putting the fix files in a bucket on the cloud, and then using a workflow task to retrieve them.  Or alternatively, copying the fix files from the cloud could become part of the RRFS-related documentation as an additional step.  These fix files DO change (quarterly?) so they need to be refreshed from time to time.

Reply:
@christopherwharrop-noaa That is fine with me.  As long as cloning of the new repos is not a significant burden for users (which you said isn't).  I'll approve.